scripts/e2e-test.sh

```diff
@@ -62,36 +62,44 @@ npm install -g $CLI_PACKAGE
 react-native init EndToEndTest --version $PACKAGE
 cd EndToEndTest
 
-case $1 in
-"--packager"*)
-  echo "Running a basic packager test"
-  # Check the packager produces a bundle (doesn't throw an error)
-  react-native bundle --platform android --dev true --entry-file index.android.js --bundle-output android-bundle.js
-  # TODO do flow check
-  ;;
-"--ios"*)
-  echo "Running an iOS app"
-  cd ios
-  # Make sure we installed local version of react-native
-  ls EndToEndTest/`basename $MARKER_IOS` > /dev/null
-  ../node_modules/react-native/packager/packager.sh --nonPersistent &
-  SERVER_PID=$!
-  # Start the app on the simulator
-  xctool -scheme EndToEndTest -sdk iphonesimulator test
-  ;;
-"--android"*)
-  echo "Running an Android app"
-  cd android
-  # Make sure we installed local version of react-native
-  ls `basename $MARKER_ANDROID` > /dev/null
-  ../node_modules/react-native/packager/packager.sh --nonPersistent &
-  SERVER_PID=$!
-  # TODO Start the app and check it renders "Welcome to React Native"
-  echo "The Android e2e test is not implemented yet" >&2
-  exit 1
-  ;;
-*)
-  echo "Please run the script with --ios, --android or --packager" >&2
-  exit 1
-  ;;
-esac
+while test $# -gt 0
```

Comment:
1. I'll add a comment where I got it from, it iterates all arguments of "scripts/e2e-test.sh" script.
In CircleCI I run 
```
./scripts/e2e-test.sh --packager --android
```
Because this would be a waste of limited CI resource:
```
./scripts/e2e-test.sh  --android
./scripts/e2e-test.sh --packager
```

The logic is to iterate all arguments and run tests for every for the same installation.

2. I added buck tests for `e2e-test --android`.
To make sure no one breaks what I did above.